test-release.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Start http server 9999
+if [ -f "http.pid" ]; then
+  kill `cat http.pid`
+  sleep 1
+fi
+echo "Starting http server on 9999..."
+python -m SimpleHTTPServer 9999 &
+echo $! > http.pid
+
+function http_test {
+  # HTTP requests
+  for i in `seq 1 10`; do
+    status_code=$(curl -v -o /dev/null -H 'Host: web' localhost:4140 2>&1 | grep '^< HTTP' | awk '{print $3}')
+    if [ -z $status_code ] || [ $status_code != "200" ]; then
+      echo "HTTP REQUEST FAILED"
+      request_failed=true
+    fi
+  done
+}
+
+function cleanup {
+  # Stop http server
+  kill `cat http.pid` && rm http.pid
+}
+
+function run_tests {
+  export LOG_LEVEL=DEBUG
+
+  ./sbt examples/acceptance-test:run &
+
+  # Wait for linkerd to initialize
+  sleep 60
+
+  # Send "load"; TODO: send actual load
+  request_failed=false
+
+  http_test
+
+  # Stop linkerd
+  curl localhost:9990/admin/shutdown
+
+  if [ $request_failed = true ]; then
+    # Report failure, leave tmp dir intact
+    echo "At least one request to linkerd failed."
+    echo " => Ensure that you have a local service running on port 9999"
+    echo " => Inspect logs at ./logs"
+    cleanup
+    exit 1
+  fi
+}
+
+echo "Running..."
```

Comment:
tioli - "Running tests..."?  This statement appears between two "Starting http server" statements so looks a little incomplete without an object
```
Starting http server on 9999...
Running...
Serving HTTP on 0.0.0.0 port 9999 ...
```